### ISSUE TITLE
Prevent start over & initial number not reached

### ISSUE THREAD

**Issue by ralphbolliger** [bfintal/Counter-Up#16]
There are two issues on my side:

```html
<span class="counter">600</span>
```

I call jQuery 2.1.4, Waypoints 3.1.1 and Counterup 1.0 (in that order). Then I call this basic script:

```javascript
$(document).ready(function() {
	$('.counter').counterUp(time:2000, delay:10);
});
```
As soon as the `.counter` element gets into viewport the script starts counting the numbers. Fine.
But when I scroll to the top of the page and scroll down again the counting starts over again (first issue). Annoying but not a problem.
The most weird thing is that the counting script is not getting to the number specified in HTML again. After page refresh it counts up to 600 as specified in HTML. Then I scroll up and down again, it starts to counf again. But it never gets up to 600. It stops at ~400. Doing the scrolling again it gets up to ~200.

Do you have any clue?

**Comment by ralphbolliger** [comment]
**Sidenote:** if I call the script without `time` and `delay` values it gets to the number specified in HTML every time.

**Comment by ralphbolliger** [comment]
Console says...
> TypeError: null is not an object (evaluating 't.data("counterup-nums").shift')
... after scrolling back to top and down again.

**Comment by digsite** [comment]
I had a very similar issue, but a different error from the console. What I found was that CounterUp 1.0 was incompatible with Waypoints 3.0+ due to changes within Waypoints. I have a [pull request](https://github.com/bfintal/Counter-Up/issues/16) that corrects this, and worked for me. Hopefully, it works for you as well!

**Comment by ciromattia** [comment]
In [my fork](https://github.com/ciromattia/jquery.counterup) I merged a bunch of PRs - it's already on bower and NPM.

This issue should be fixed with Waypoints-4.0.0 support, please check it out and tell me if it works.
Thanks!

**Comment by hipchecker** [comment]
@ralphbolliger Agreed that it's annoying that the counter loads every time the element is scrolled into view. 
@ciromattia Did any of your merges include a way to only load the counter animation on first scroll, but not again until page reload?

**Comment by ciromattia** [comment]
@hipchecker this is already happening AFAIK. Do you have this same issue even with my version?

**Comment by hipchecker** [comment]
@ciromattia I moved this conversation to an issue on your fork https://github.com/ciromattia/jquery.counterup/issues/5
